src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+const tableRows = [...tbody.rows];
+
+thead.addEventListener('click', () => {
+  const onlyNumbers = string => +string.replace(/[$,]/g, '');
```

Comment:
Consider using a more descriptive name for the `onlyNumbers` function, such as `extractNumbers`.